jobcontroller/jobobserver.py

```diff
@@ -0,0 +1,6 @@
+class JobObserver:
```

Comment:
I am pointing this out on this line, but can we snake cakse the file names, I can only read this as jo bob server dot py lol